main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+// This will be loaded from env later I'm assuming. For now just returns what port is used on localhost.
+const HOST = "127.0.0.1"
+const PORT = "8080"
+
+var ADDRESS = fmt.Sprintf("%s:%s", HOST, PORT)
+
+func main() {
+	fmt.Println("serving on address: ", ADDRESS)
+	router := gin.Default()
+	router.GET("/health", getHealth)
+	http.ListenAndServe(ADDRESS, router)
+}
+
+func getHealth(c *gin.Context) {
+	c.IndentedJSON(http.StatusOK, gin.H{
+		"status": 200, "serving": ADDRESS,
```

Comment:
```suggestion
		"status": "serving"
```

the HTTP 200 is returned already outside of the JSON just as part of the normal HTTP spec.  It's redundant to have it here.  The common practice here is to have this be human-readable for the healthcheck, so "status: OK" or "status: serving" or something. 

And for security reasons, don't expose the serving address in the API.  It can leak details that malicious attackers would use.

Reply:
sorry, missed this one before